.github/matrix-jvm-tests.json

```diff
@@ -13,8 +13,8 @@
   "os-name": "ubuntu-latest"
 }
 , {
-  "name": "19",
-  "java-version": 19,
+  "name": "20",
+  "java-version": 20,
```

Comment:
@gsmet do you rremember if there was a reason we hadn't already done this?

Reply:
All I could find was #31063, an issue with Gradle. But we are running Gradle tests only on 11 anyway.

Reply:
Seems reasonable, but I want to avoid in surprises, which is why I want @gsmet's input.